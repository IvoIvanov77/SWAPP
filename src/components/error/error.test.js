import React from 'react';
import { shallow } from 'enzyme';
import ProcessError from './error.container';
import { MockedProvider } from '@apollo/react-testing';

it('should render ProcessError component', () => {
  expect(
    shallow(
      <MockedProvider>
        <ProcessError />
      </MockedProvider>,
    ),
  ).toMatchSnapshot();
});
