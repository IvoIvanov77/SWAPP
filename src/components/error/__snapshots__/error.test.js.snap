// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ProcessError component 1`] = `ShallowWrapper {}`;
